src/components/common/ColonyActions/ColonyActions.tsx

```diff
@@ -223,6 +223,19 @@ const ColonyActions = (/* { ethDomainId }: Props */) => {
         )}
         text="Test Create Domain"
       />
+      <ActionButton
+        submit={ActionTypes.ACTION_UNLOCK_TOKEN}
+        error={ActionTypes.ACTION_UNLOCK_TOKEN_ERROR}
+        success={ActionTypes.ACTION_UNLOCK_TOKEN_SUCCESS}
```

Comment:
Wasn't there a PR that removed the need to add all 3 actions specifically ?

You would just add the the base submit action name, and the `ActionButton` refactor would generate it's own ones for error and success ?

Reply:
It's in #261 and I'm pretty sure I saw Armando using it already in his UAC porting work. I'm hesitant to track changing it here since that'll be removed sooner or later